packaging/centreon-broker-cbd-debuginfo.yaml

```diff
@@ -0,0 +1,43 @@
+name: "centreon-broker-cbd-debuginfo"
+arch: "${ARCH}"
+platform: "linux"
+version_schema: "none"
+version: "${VERSION}"
+release: "${RELEASE}${DIST}"
+section: "default"
+priority: "optional"
+maintainer: "Centreon <contact@centreon.com>"
+description: |
+  Debuginfo package for centreon-broker-cbd.
+  Commit: @COMMIT_HASH@
+vendor: "Centreon"
+homepage: "https://centreon.com"
+license: "Apache-2.0"
```

Comment:
Isn't it required to add the license content somewhere ?

Reply:
License should be at the root directory of the repo I guess
But it must be managed in an other PR